fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/domain/ChainedSearchParam.java

```diff
@@ -85,8 +90,13 @@ public ChainedSearchParam(String rootResourceType, String name, QueryParameter q
             msp.visit(currentSubQuery, visitor);
         } else if (currentParm.getType() == Type.COMPOSITE) {
             visitor.addCompositeParam(currentSubQuery, currentParm);
-        } else if (currentParm.isCanonical()) {
+        } else if (currentParm.isCanonical() || PROFILE.equals(currentParm.getCode()) ||
```

Comment:
I just want to double-check that the boolean expression you have here is the one you want. The && expression has parens which are redundant...but I don't know what the intended logic is supposed to be. The expression you have is:
```
   A || B || C && D
```

Reply:
yeah, that is the expression I want